interface/modules/zend_modules/module/Syndromicsurveillance/view/layout/layout.phtml

```diff
@@ -31,9 +31,9 @@
         <?php echo $this->headScript()->prependFile($this->basePath() . '/js/scripts/syndromicsurveillance.js')
         ->prependFile($this->basePath() . '/js/application/sendTo.js')
         ->prependFile($this->basePath() . '/js/application/common.js')
-        ->prependFile($this->basePath() . '/js/carecoordination/jquery-ui.custom.js')
-        ->prependFile($this->basePath() . '/js/carecoordination/jquery.ui.datepicker.js')
-        ->prependFile($this->basePath() . '/js/carecoordination/jquery.ui.core.js')
+        ->prependFile($this->basePath() . '/js/lib/jquery-ui.custom.js')
+        ->prependFile($this->basePath() . '/js/lib/jquery.ui.datepicker.js')
```

Comment:
Testing: We should at least point to /public/assets in this file so we can get rid of the excess files.